offer-manager-domain/src/main/groovy/life/qbic/business/products/Product.groovy

```diff
@@ -59,7 +59,7 @@ class Product {
 
     /**
      * Calculates the checksum for the product
-     * @return
+     * @return a string containing the checksum for this product
```

Comment:
How is it computed (variables?) what type of checksum is it?

Reply:
👍  exactly, that was what I meant. This should be added